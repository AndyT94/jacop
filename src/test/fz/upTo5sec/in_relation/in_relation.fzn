array [1..3] of var int: v1:: output_array([1..3]) = [5,3,3];
solve  satisfy;
