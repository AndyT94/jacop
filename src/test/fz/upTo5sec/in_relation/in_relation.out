v1 = array1d(1..3, [5, 3, 3]);
----------